### ISSUE TITLE
Documentation on "how to start" missing

### ISSUE THREAD

**Issue by om26er** [HH95/HH---POS-Accounting-and-ERP-Software#9]
How to start this project ? I just cloned the source but haven't found any documentation on how to launch the app. I need instructions for a Linux based environment.

Sorry for posting the question here, I didn't know of a way to contact/ask questions.

**Comment by clefavelli** [comment]
Am also wondering , are we supposed to chew the code ( just kidding ) , did you get a way ?

**Comment by om26er** [comment]
> did you get a way ?

no, we ended up writing something from scratch using PySide2 ;-)

**Comment by clefavelli** [comment]
To run this project you mean, o did you have to do a new project from scratch

**Comment by clefavelli** [comment]
Is it something you can share, I have to set up a POS urgently? I also want to dockerize this project if am allowed to.

**Comment by om26er** [comment]
> Is it something you can share, I have to set up a POS urgently? I also want to dockerize this project if am allowed to.

That is unfortunately proprietary and something we haven't decided to open-source just yet.

**Comment by clefavelli** [comment]
So the guy left the documentation on starting out intentionally?
Or what?
